test/shopify-cli/theme/dev_server/integration_test.rb

```diff
@@ -64,7 +64,6 @@ def test_proxy_to_sfr
         end
 
         def test_uploads_files_on_boot
-          skip
```

Comment:
unresolved for now @karreiro

Reply:
Yes, let's remove that `skip`; hopefully, the other changes have stabilized this test :) or even we will get more information if we notice it failing again.